nn/client.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+"""
+    Reynir: Natural language processing for Icelandic
+
+    Neural Network Query Client
+
+    Copyright (C) 2020 Miðeind ehf.
+
+       This program is free software: you can redistribute it and/or modify
+       it under the terms of the GNU General Public License as published by
+       the Free Software Foundation, either version 3 of the License, or
+       (at your option) any later version.
+       This program is distributed in the hope that it will be useful,
+       but WITHOUT ANY WARRANTY; without even the implied warranty of
+       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+       GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/.
+
+    This module implements a client that connects to a middleware
+    neural network server (see nnserver/nnserver.py), which in turn
+    connects to a TensorFlow model server.
+
+"""
+
+import json
+
+from flask import abort
+import requests
+
+from settings import Settings
+
+
+class ApiClient:
+
```

Comment:
Add a short documentation string describing the purpose of this class and any special considerations